lib/WorkflowRaredisease.groovy

```diff
@@ -35,6 +35,20 @@ class WorkflowRaredisease {
         return outfile
     }
 
+    //
+    // Create a bed file which includes the name of variant caller from fai
+    //
+    public static String makeBedWithVariantCallerInfo(fai, parent_dir, varcaller) {
+        def outfile = new File(parent_dir + '/' + varcaller +'.bed')
+        def writer  = outfile.newWriter()
+        fai.eachLine { line ->
+            def split_str = line.tokenize("\t")
+            writer << [split_str[0],"1",split_str[1],varcaller].join("\t") + "\n"
```

Comment:
```suggestion
            writer << [split_str[0],"0",split_str[1],varcaller].join("\t") + "\n"
```
The bed format is zero based. However I imagine that it will make very little difference here ;)